pkg/snapshot/filesystem.go

```diff
@@ -1101,3 +1113,22 @@ func IsFileSystemMinioDisabled(kotsadmNamespace string) (bool, error) {
 
 	return false, nil
 }
+
+func IsMinIOImagePresent(client kubernetes.Interface) (bool, error) {
```

Comment:
Will do lookup based on the latest-version ([MinioImage](https://github.com/replicatedhq/kots/pull/2523/files#diff-76bce06844033f37b1295a290dcd94fdfbf2e8e6a4a737210b13e8bc4a5bbc4cR13)) is a good idea so we dont look for the older image versions if there is any.